lib/controllers/getTarball.js

```diff
@@ -4,27 +4,43 @@ function getTarball(respond, route, unpm) {
   var version = route.params.version
   var name = route.params.name
   var started = false
+  var getTarballAsync = unpm.Package.getTarballAsync
+  var getTarballSync = unpm.Package.getTarball
```

Comment:
@hayes Are you referring to simply renaming the imports to different variable names? Or are you proposing changing the actual imported functions to be `unpm.Package.getTarballStreamAsync` and `unpm.Package.getTarballSync` rather than `unpm.Package.getTarballAsync` and `unpm.Package.getTarball`?